Makefile

```diff
@@ -2,6 +2,7 @@ COMPONENTS = daemon controller server operator
 
 # vim: noexpandtab ts=8
 export GOPATH=$(shell echo $${GOPATH:-$$HOME/go})
+export GOFLAGS="-tags=containers_image_openpgp"
```

Comment:
we may want this specifically in ./hack/build-go.sh (at the end, filtering for machine-config-controller)